docs/UserGuide.md

```diff
@@ -135,6 +137,26 @@ Examples:
 * `list c` followed by `view c 1` views the first contact in the companies list:
   ![show the information of the first company](images/viewCompany.png)
 
+### Editing a person's contact: `edit p`
+
+Edit the information of a person's contact
+
+Format: `edit p [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]`
```

Comment:
Is this missing the `INDEX` parameter?

Reply:
Yes, I forgot the INDEX field. Also fixed.

Reply:
Same for the following issues you raised. Thanks for pointing out.